ui-tests/cypress/integration/components/micro-frontends-and-widgets.spec.js

```diff
@@ -20,7 +22,16 @@ describe('Microfrontends and Widgets', () => {
     cy.kcLogout();
   });
 
+  const selectPageFromSidebar = (pageOpen = PAGE) => {
+    const currentPageContent = currentPage.getContent();
+    currentPageContent.getSidebarTab('Page Tree').click();
+    cy.wait(3000);
+    currentPageContent.getPageTreeItem(pageOpen.title).click({force: true});
```

Comment:
Is `force:true` really needed??

Reply:
hi @g-di-martino this was already an existing function that I transferred from page designer test block to the entire MFE/widgets test block. As I recalled, `force: true` was added because before I was using without that option, it won't click the given page (at least in my end, though) - so I added that option and it works.

Reply:
I just retested it without the option and it works this time. maybe it was needed when it isn't using page object pattern. not quite sure though. pls re-review

Reply:
It shouldn't be needed when the element is clearly identified (in which the page object pattern helps a lot) and application is in the expected state at that point (which is easier to understand with the page object pattern) :wink: